data/external/data_description.md

```diff
@@ -0,0 +1,33 @@
+`orders` (3.4m rows, 206k users):
```

Comment:
good comprehensive coverage of variables being used and their description

Reply:
would you be able to do the same thing if there are over 100 features for example?

Reply:
ahh this file actually gets directly downloaded and unpacked with the data download - I did not make it